demoOfControlFlow.html

```diff
@@ -47,12 +47,14 @@ <h5 class="mb-1">Text will be copied here</h5>
                 log.textContent = e.target.value;
             }
 
-            function updateContainer() {   //I CANNOT FOR THe life OF ME GET THIS TO WORK!!!! IM GOING BACK TO THIS TODAY LATER!!
+            function updateContainer() {
                 if(checkedBoxRecord.checked == false) {
```

Comment:
This is good, but I would have liked to see more examples. For example, using `if... else`, switch statements, and ternary operators.

Minor comment, you want to use `===`. If you only use `==`, the following will evaluate to true:

```
if('false' == false)
```